worldedit-core/src/main/java/com/sk89q/worldedit/world/storage/ChunkStore.java

```diff
@@ -78,7 +79,32 @@ public static BlockVector2D toChunk(Vector position) {
      * @throws IOException thrown on I/O error
      */
     public Chunk getChunk(Vector2D position, World world) throws DataException, IOException {
-        CompoundTag tag = getChunkTag(position, world);
+        CompoundTag rootTag = getChunkTag(position, world);
+
+        Map<String, Tag> children = rootTag.getValue();
+        CompoundTag tag = null;
+
+        // Find Level tag
+        for (Map.Entry<String, Tag> entry : children.entrySet()) {
+            if (entry.getKey().equals("Level")) {
+                if (entry.getValue() instanceof CompoundTag) {
+                    tag = (CompoundTag) entry.getValue();
+                    break;
+                } else {
+                    throw new ChunkStoreException("CompoundTag expected for 'Level'; got " + entry.getValue().getClass().getName());
+                }
+            }
+        }
+
+        if (tag == null) {
+            throw new ChunkStoreException("Missing root 'Level' tag");
+        }
+
+        int dataVersion = rootTag.getInt("DataVersion");
+        if (dataVersion >= 1519) { // data version of Minecraft 1.13
```

Comment:
Perhaps extract `1519` to a constant named `DATA_VERSION_MC_1_13`?

Then you won't need the comment, and it'll be easier to read in-line.

Reply:
Ok I changed that.